content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -571,3 +565,19 @@ example lbs=2014.9 1578096000000000000
 ```
 {{% /flex-content %}}
 {{< /flex >}}
+
+## Flux
+
+Use the [csv.from()](/influxdb/v2.0/reference/flux/stdlib/csv/from/) and [to()](/influxdb/v2.0/reference/flux/stdlib/built-in/outputs/to/) Flux functions to write an annotated CSV to the bucket of your choice.
+
+{{< youtube wPKZ9i0DulQ >}}
+
+The experimental [csv.from()](/influxdb/v2.0/reference/flux/stdlib/csv/from/) function lets you write CSV from a URL.
+The example below writes [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data) to an example `noaa` bucket in an example organization:
+
+```js
+import "experimental/csv"
+
+csv.from(url: "https://influx-testdata.s3.amazonaws.com/noaa.csv")
+  |> to(bucket: "noaa", org: "example-org")
+```
```

Comment:
We should probably add a note about required columns when using Flux to write to InfluxDB.

~~~suggestion
```js
import "experimental/csv"

csv.from(url: "https://influx-testdata.s3.amazonaws.com/noaa.csv")
  |> to(bucket: "noaa", org: "example-org")
```

{{% note %}}
#### Required columns
To write data to InfluxDB with Flux, data must include the following columns:

- `_time`
- `_measurement`
- `_field`
- `_value`
{{% /note %}}
~~~